steely/plugins/url_expander.py

```diff
@@ -0,0 +1,34 @@
+from urllib.request import Request, urlopen
+import json
+
+
+def expand(url):
+    API_ADDRESS = 'http://x.datasig.io/short?url={}'
```

Comment:
maybe make this a global? because does this cause an extra allocation on the stack? not sure